scripts/pyannote-face.py

```diff
@@ -371,20 +378,23 @@ def make_frame(t):
                         (pt1[0], pt1[1] - 7), cv2.FONT_HERSHEY_SIMPLEX,
                         0.5, (255, 0, 0), 1, 8, False)
 
-            # Draw nose
-            if landmark:
-                points = landmarks[i][0].parts()
-                pt1 = (int(points[27, 0]), int(points[27, 1]))
-                pt2 = (int(points[33, 0]), int(points[33, 1]))
-                cv2.line(frame, pt1, pt2, color, 1)
+            # Draw face landmarks
+            # Check if a landmark exists in index i
+            if landmark and (i < len(landmarks)):
```

Comment:
If `len(landmarks) >= i` happens, it probably means that there was a bug in the facial landmark detection.
We should probably fix the landmark detection mode instead...